### ISSUE TITLE
It should be possible to define multiple mongo-urls for separate servers

### ISSUE THREAD

**Issue by AndreasGalster** [zodern/meteor-up#158]
We intend to use a separate mongo-url for our staging server but it looks like right now we can only share one mongo-url via the env variables. It would be very nice to have to be able to define the url per server.

**Comment by madushan1000** [comment]
There is an (planned) option to specify server specific env vars trough the currently empty blocks like `one: { env: { SOMETHING: "something" } }`. But it's not implemented yet.

``` js
servers: {
      one: {}, two: {}, three: {} //list of servers to deploy, from the 'servers' list
},
```

**Comment by rclai** [comment]
It would be nice to also be able to set other arbitrary environment variables per server so that they can be accessed via `process.env.MY_VAR`.

**Comment by zodern** [comment]
This will be available in Mup 1.3.
